<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GreenmileWidget</class>
 <widget class="QWidget" name="GreenmileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="routesNotUploadedBox">
         <property name="text">
          <string>Routes not uploaded</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="thursdayGMButton">
         <property name="text">
          <string>Thursday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="mondayGMButton">
         <property name="text">
          <string>Monday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="tuesdayGMButton">
         <property name="text">
          <string>Tuesday</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QCheckBox" name="routesMisassignedBox">
         <property name="text">
          <string>Routes Misassigned</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="saturdayGMButton">
         <property name="text">
          <string>Saturday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="fridayGMButton">
         <property name="text">
          <string>Firday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="wednesdayGMButton">
         <property name="text">
          <string>Wednesday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="sundayGMButton">
         <property name="text">
          <string>Sunday</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="7">
        <widget class="QPushButton" name="todayGMButton">
         <property name="text">
          <string>Today</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Please Wait</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="gmAbortButton">
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="gmRunReportAgain">
         <property name="text">
          <string>Run Report Again</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
